packages/blocks/src/blocks.spec.ts

```diff
@@ -4,38 +4,34 @@ import { chain } from 'ramda';
 import {
```

Comment:
only thing that changed about these tests were moving out element tests and a slight reorganization to better match the source blocks file.